cmd/crc/cmd/stop.go

```diff
@@ -32,6 +32,10 @@ func stopMachine(client machine.Client, interactive, force bool) (bool, error) {
 		return false, err
 	}
 
+	if running, _ := client.IsRunning(); !running {
+		return false, fmt.Errorf("Cluster is already stopped")
```

Comment:
what does the HTTP API reply in this case:
Will it also error or success with this message?

Because, I do not believe this should be an error, but rather info?

Reply:
Agree, if you are running stop on an stopped cluster then it just provide info and not error. But then same with other API which query when crc vm is not exist, like ip, status .. etc. `checkIfMachineMissing` always return error.

Reply:
> what does the HTTP API reply in this case:
Will it also error or success with this message?

Error, HTTP api will respond with a 500 internal server error and no body

Reply:
I just wonder how a stopping a stopped VM can result in an error; does starting a started VM also do this? what about deleting a deleted VM?

Reply:
As of now `starting` an already running VM just print out all the details about openshift user messages. if we `delete` a deleted machine it through an `machine doesn't exist error`

Reply:
Looking at the implementation of `IsRunning`, the state of the VM is not running but it can be anything like following. So we shouldn't say it is stopped but may be `not running` ?

```
	"Paused",
	"Saved",
	"Stopped",
	"Stopping",
	"Starting",
	"Error",
	"Timeout",
```

Reply:
> Error, HTTP api will respond with a 500 internal server error and no body.

If you put a similar check in `pkg/crc/machine/stop.go` rather than in `cmd/`,  this is going to  solve this both for HTTP and command line.

Reply:
@gbraad @cfergeau Done, moved the check to `pkg/crc/machine/stop.go`